Description: <patch for gsetting>
 .
 .
 gsettings-desktop-schemas (3.24.1-1blankon9) uluwatu; urgency=medium
 .
   * blankon-gsetting
Author: AnTo SamalonA <ansamsoftdev@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Last-Update: 2017-11-19

--- gsettings-desktop-schemas-3.24.1.orig/schemas/gsettings-desktop-schemas.convert
+++ gsettings-desktop-schemas-3.24.1/schemas/gsettings-desktop-schemas.convert
@@ -12,6 +12,9 @@ autorun-x-content-start-app = /apps/naut
 autorun-x-content-ignore = /apps/nautilus/preferences/media_autorun_x_content_ignore
 autorun-x-content-open-folder = /apps/nautilus/preferences/media_autorun_x_content_open_folder
 
+[org.gnome.desktop.background]
+picture-uri = /desktop/gnome/background/picture_filename
+
 # From org.gnome.desktop.default-applications.gschema.xml
 [org.gnome.desktop.default-applications.office.calendar]
 exec = /desktop/gnome/applications/calendar/exec
--- gsettings-desktop-schemas-3.24.1.orig/schemas/org.gnome.desktop.background.gschema.xml.in
+++ gsettings-desktop-schemas-3.24.1/schemas/org.gnome.desktop.background.gschema.xml.in
@@ -10,7 +10,7 @@
       </description>
     </key>
     <key name="picture-uri" type="s">
-      <default>'file://@datadir@/backgrounds/gnome/adwaita-timed.xml'</default>
+      <default>'file://@datadir@/gnome-background-properties/blankon-wallpapers.xml'</default>
       <summary>Picture URI</summary>
       <description>
         URI to use for the background image. Note that the backend only supports
--- gsettings-desktop-schemas-3.24.1.orig/schemas/org.gnome.desktop.interface.gschema.xml.in
+++ gsettings-desktop-schemas-3.24.1/schemas/org.gnome.desktop.interface.gschema.xml.in
@@ -83,14 +83,14 @@
       </description>
     </key>
     <key name="icon-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Tebu-Flat'</default>
       <summary>Icon Theme</summary>
       <description>
         Icon theme to use for the panel, nautilus etc.
       </description>
     </key>
     <key name="gtk-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Bromo'</default>
       <summary>Gtk+ Theme</summary>
       <description>
         Basename of the default theme used by gtk+.
@@ -104,7 +104,7 @@
       </description>
     </key>
     <key name="font-name" type="s">
-      <default>'Cantarell 11'</default>
+      <default>'Aurulent Sans 10'</default>
       <summary>Default font</summary>
       <description>
         Name of the default font used by gtk+.
@@ -148,14 +148,14 @@
       </description>
     </key>
     <key name="document-font-name" type="s">
-      <default>'Sans 11'</default>
+      <default>'Sans 10'</default>
       <summary>Document font</summary>
       <description>
         Name of the default font used for reading documents.
       </description>
     </key>
     <key name="monospace-font-name" type="s">
-      <default>'Monospace 11'</default>
+      <default>'Monospace 10'</default>
       <summary>Monospace font</summary>
       <description>
         Name of a monospaced (fixed-width) font for use in locations like
@@ -170,7 +170,7 @@
       </description>
     </key>
     <key name="cursor-theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'DMZ-White'</default>
       <summary>Cursor theme</summary>
       <description>Cursor theme name. Used only by Xservers that support the Xcursor extension.</description>
     </key>
--- gsettings-desktop-schemas-3.24.1.orig/schemas/org.gnome.desktop.sound.gschema.xml.in
+++ gsettings-desktop-schemas-3.24.1/schemas/org.gnome.desktop.sound.gschema.xml.in
@@ -7,7 +7,7 @@
       <description>Whether to play sounds on user events.</description>
     </key>
     <key name="theme-name" type="s">
-      <default>'freedesktop'</default>
+      <default>'blankon'</default>
       <summary>Sound theme name</summary>
       <description>The XDG sound theme to use for event sounds.</description>
     </key>
--- gsettings-desktop-schemas-3.24.1.orig/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in
+++ gsettings-desktop-schemas-3.24.1/schemas/org.gnome.desktop.wm.preferences.gschema.xml.in
@@ -25,7 +25,7 @@
        </description>
     </key>
     <key name="button-layout" type="s">
-      <default>'appmenu:close'</default>
+      <default>':minimize,maximize,close'</default>
       <summary>Arrangement of buttons on the titlebar</summary>
       <description>
         Arrangement of buttons on the titlebar. The value should be a string,
@@ -174,7 +174,7 @@
       </description>
     </key>
     <key name="theme" type="s">
-      <default>'Adwaita'</default>
+      <default>'Bromo'</default>
       <summary>Current theme</summary>
       <description>
         The theme determines the appearance of window borders, titlebar,
@@ -192,7 +192,7 @@
       </description>
     </key>
     <key name="titlebar-font" type="s">
-      <default>'Cantarell Bold 11'</default>
+      <default>'Aurulent Sans Bold 10'</default>
       <summary>Window title font</summary>
       <description>
         A font description string describing a font for window titlebars.
